src/rest/routes/analytics/index.ts

```diff
@@ -0,0 +1,70 @@
+import { errorWrapper } from "~/src/rest/helper";
+import { Request, Response } from "express";
+import { sendAnalytics } from "utils/analytics-client";
+
+interface TrackUiEventProps {
```

Comment:
/nit
Can reuse the existing rest-interface/index.ts to share the typing. <-- but only `type` is supported.

Reply:
waitasec.... rest-interface is in spa, while this code is in the backend...

Reply:
waitasec.... wtf